.goreleaser.yml

```diff
@@ -0,0 +1,49 @@
+before:
+  hooks:
+    # you may remove this if you don't use vgo
+    - go mod tidy
+builds:
+  -
+    binary: biscuit
+    goos:
+      - darwin
+      - linux
+      - windows
+    goarch:
+      - amd64
+      - arm64
+      - 386
+    ldflags:
+      - -s -w
+      - -X main.Version={{.Version}}
+archives:
+-
+  name_template: "{{.ProjectName}}_{{.Version}}_{{.Os}}-{{.Arch}}"
+  replacements:
+    darwin: macOS
+    linux: Linux
+    windows: Windows
+    amd64: 64bit
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
+changelog:
+  sort: asc
+  filters:
+    exclude:
+    - '^docs:'
+    - '^test:'
+
+dockers:
+  -
+    goos: linux
+    goarch: amd64
+    ids:
+    - biscuit
+    image_templates:
+    - "ghcr.io/dcoker/biscuit:latest"
+    - "ghcr.io/dcoker/biscuit:{{ .Tag }}"
+    dockerfile: .goreleaser.Dockerfile
+    build_flag_templates:
+    - "--pull"
```

Comment:
This might need a token to push to Github container registry. I don't think the token that goes with the repo will allow for GHCR.